package.json

```
@@ -3,13 +3,13 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "generate": "./scripts/generate.sh",
+    "generate": "rm -rf ./seam && node ./scripts/generate.js",
```

Move the del step into the script, e.g., `npm I -D del`
```suggestion
    "generate": "node ./generate.js"
```

---

@razor-x did you mean that we should install `del` package with `npm I -D del ` and use it to delete the directory or add `del` script like `"del": "rm -rf ./seam"` and use it in ""generate": "npm run del && node ./scripts/generate.js" ?

---

Yea, maybe

```
await del('./seam')
await main()
```

Actually, let's make this a bit safer so it can't delete the wrong folder

```
import { fileURLToPath } from 'node:url'
await del(fileURLToPath(new URL('./dist', import.meta.url)))
```

---

Did you mean `await del(fileURLToPath(new URL('./seam', import.meta.url)))`? 
Or we need to delete `./dist` folder as well?

---

@razor-x Could you take a look at commit mentioned above if I got it right?